src/components/categoryitems.main.jsx

```diff
@@ -167,9 +279,15 @@ class CategoryItemsMain extends React.Component {
                 </div>
                 <SearchFacetNavigationMain productData={products} titleString={categoryModelIdString} />
                 <div className="products-container">
+                  <div className="sort-product-menu">
```

Comment:
I'd prefer if this was a separate component. Was there a reason for including the logic at this level?

Reply:
I'll try to do it like a separate component